src/test/java/uk/gov/hmcts/reform/sscs/robotics/json/RoboticsJsonValidatorTest.java

```diff
@@ -92,14 +92,14 @@ public void givenInvalidInputForHearingType_throwExceptionWhenValidatingAgainstS
     @Test(expected = RoboticsValidationException.class)
     public void givenOralInputForLanguageInterpreterWithNoHearingRequestParty_throwExceptionWhenValidatingAgainstSchema() throws ValidationException, IOException {
         jsonData = updateEmbeddedProperty(jsonData.toString(), "Oral", "hearingType");
-        jsonData = removeProperty(jsonData.toString(), "hearingRequestParty");
+        jsonData = removeProperty(jsonData.toString());
```

Comment:
Is this actually removing the hearingRequestParty from the json now as part of the test? Don't understand what this test does now it's been changed.